pyiron/lammps/base.py

```diff
@@ -915,17 +917,34 @@ def collect_dump_file(self, file_name="dump.out", cwd=None):
         )
         output["forces"] = np.matmul(forces, rotation_lammps2orig)
 
+        if 'f_mean_forces[1]' in content[0].keys():
+            forces = np.array(
+                [np.stack((cc["f_mean_forces[1]"], cc["f_mean_forces[2]"], cc["f_mean_forces[3]"]), axis=-1) for cc in content]
+            )
+            output["mean_forces"] = np.matmul(forces, rotation_lammps2orig)
```

Comment:
Since we're abandoning 2.7, I switched over to using the `A @ B` syntax for matrix multiplication. This is supported from, I think, 3.5 onwards.

I don't insist that we do this, but if you also think it's nice now would be a good time to get rid of a few `matmul` calls.

Reply:
well since it looks like it's unified with `matmul`, I'll keep this one here for now, but for other cases I'll certainly take `A @ B` into consideration :)